src/main/scala/view/scalaFX/FXControllers/BaseAppController.scala

```diff
@@ -142,18 +145,27 @@ class BaseAppController(
   /** Handler of Start button click */
   def startSimulation(): Unit = {
     startButton.setVisible(false)
+    informationPanel.visible = false
     Controller.startSimulation()
   }
 
   /** Handler of Summer button click */
   def setEnvironmentSummer(): Unit = {
     Controller.setSummerClimate()
+    summerButton.styleClass -= "button-clickable"
```

Comment:
Volendo potresti rifattorizzare in un metodo che attiva un bottone e disattiva l'altro da usare poi sia in `setEnvironmentSummer()` che in `setEnvironmentWinter()`